requirements.txt

```
@@ -1,9 +1,10 @@
 colorama>=0.4.5
 websockets>=10.3
 PyYAML>=6.0
-jellyfish>=0.9.0
+jellyfish>=0.11.0; python_version >= '3.10'
+jellyfish>=0.10.0; python_version <= '3.9'
```

I think this was fixed in https://github.com/jamesturk/jellyfish/issues/179
```suggestion
jellyfish>=0.11.0
```